PR fixing bug #269: Common-envelope event: Legacy bug in eccentric RLOF leading to a CEE

avigna opened the pull request.

avigna committed:
Adding testfile to CEE fix branch

avigna committed:
Added a getter function for periastron.

avigna committed:
Changed everything within the ResolveCommonEnvelopeEvent function to use periastron in Rsol units for calculations. Separation values are written in Rsol to the output and final separation in AU (default COMPAS units).

avigna committed:
Deleted function calls for tidal timescales within the CEE funcion

avigna committed:
Mostly editing units in CE printing

avigna committed:
Removed printed statements.

avigna committed:
Added some getters and cleaned the function a bit.

jeffriley reviewed (changes requested):
The changes are good, except for the following:

(a) I don't see updates to the change log and version number in constants.h.  You can put them in, and if there are conflicts when merging we can make them right then

(b) what is the test file?

avigna committed:
Update constants.h

Added changes to log and version number. Leaving both numbers as placeholders as I know I am not up to date with Ilya's recent changes.

avigna commented:
Thanks Jeff!
> (a) I don't see updates to the change log and version number in constants.h. You can put them in, and if there are conflicts when merging we can make them right then
I added the strings. I unfortunately lost track of Ilya's changes and not sure what number it should be (I probably didn't fetched to the right dev branc when I started working on this bug).

> (b) what is the test file?
It was a random txt file I made just to be sure I was pushing to the right place. I should have deleted it (and assumed I did as I do not see it in the changed files).

jeffriley committed:
Merge branch 'dev' into CEE_finalSeparation_267

jeffriley committed:
Delete test.txt

jeffriley reviewed (approved):
Merge conflicts resolved.
